### ISSUE TITLE
The keyboard trap occurs on the "Notebook" screen when the keyboard focus enters the editor block and cannot move out of the editor box.: A11y_SQL Azure Data Studio_NoteBook_Keyboard

### ISSUE THREAD

**Issue by Mana-swi** [microsoft/azuredatastudio#26420]
### Tags:
#A11yTCS;#A11yWCAG;#WCAG2.1.2;#No Keyboard Trap;#Keyboard;#A11ySev2;#Win32;#DesktopApp;#BM_SQL Azure Data Studio_Win32_Sep25;#SQL Azure Data Studio;
**"[[Check out Accessibility Insights!]](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)) - Identify accessibility bugs before check-in and make bug fixing faster and easier."**
### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.52.0 (system setup)
VS Code: 1.82.0
Electron: 30.5.1
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22621
### Repro Steps:
1. Launch Azure Data Studio.
2. Connect to server.
3. In the Connections pane, right-click on a server connection. 
4. Then select 'Manage' from the context menu to view the dashboard.
5. Tab till show/hide panel button and inspect it.
7. Observe that The keyboard trap occurs on the "Notebook" screen when the keyboard focus enters the editor block and cannot move out of the editor box.
### Actual:
The keyboard trap occurs on the "Notebook" screen when the keyboard focus enters the editor block and cannot move out of the editor box. With F6 key the focus is moving out of the entire screen on bottom task bar and user need to navigate all the screen to get to the controls in the Notebook screen.

Note:
Similar issue is observed throughout the application for editor block.

### Expected:
The keyboard trap should not occur when the keyboard focus enters the editor block and there should be a shortcut key provided to users to move out of the editor box and access the other controls of the screen.

### User Impact:
Keyboard users will not be able to move out of the editor block once they entered blocking them to access the rest controls of the screen.
### Recommendations:
 [MAS Problems and Suggested Fixes](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fmicrosoft.sharepoint.com%2fsites%2faccessibility%2fLists%2fMASProblemsAndSuggestedFixes%2fReviewed%2520Items.aspx&data=04%7c01%7cv-prajadhav%40microsoft.com%7cf642cb2fc53d4c4be38408d9d97cf216%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637779952946112692%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000&sdata=avswAOFlXEiWy%2fRT%2f%2fPGnurQzMGOUdSW7Z5Xp8ZioiQ%3D&reserved=0)
### MAS Reference:

- [MAS 2.1.2 – No Keyboard Trap](https://aka.ms/MAS2.1.2)
- [MAS 2.1.1: Keyboard Guide](https://microsoft.sharepoint.com/sites/accessibility/SitePages/MAS-2.1.1-Keyboard-Guide.aspx)

![Image](https://github.com/user-attachments/assets/731c146a-433d-4571-bdcb-24462795a4d9)
